packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx

```diff
@@ -75,6 +85,14 @@ const getInitialProcessPayload = ({
             subscription_id: subscriptionId,
         };
     }
+    if (productId && prefix && prefixlen && prefix_min) {
```

Comment:
Isn't this unreachable code? the if on line 78 above `if (productId) {` would trigger true before this if is reached